server/src/game/Room.ts

```
@@ -319,8 +380,8 @@ class SingleplayerRoom extends Room {
 
     // check anticheat and submit score
     if (socket) {
-      if (performAnticheatCheck(data.actionRecords).ok) {
-        submitSingleplayerGame(data, socket);
+      if (true) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)